VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -88,7 +91,28 @@ export class AppComponent implements OnInit, OnDestroy {
         return window.location.pathname.includes(pageUrls.EJudSignIn) || window.location.pathname.includes(pageUrls.VHSignIn);
     }
 
+    getUserName(): any {
+        return this.username;
+    }
+
+    setVisibility(value: boolean) {
```

Comment:
since this is a handler for the event emitter, label this as `setBannerVisibility` so it's clear it's the banner and not another component

Reply:
done